<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QWidget" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="groups_listWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>211</width>
     <height>551</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="groups_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Groups</string>
   </property>
  </widget>
  <widget class="QListWidget" name="contributors_listWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>611</width>
     <height>551</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="contributors_label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>181</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Contributors</string>
   </property>
  </widget>
  <widget class="QPushButton" name="draw_pushButton">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>50</y>
     <width>113</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Draw</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
